src/providers/twitch/IrcMessageHandler.cpp

```diff
@@ -19,6 +19,38 @@
 
 #include <unordered_set>
 
+namespace {
+using namespace chatterino;
+MessagePtr generateBannedMessage(bool confirmedBan)
+{
+    const auto linkColor = MessageColor(MessageColor::Link);
+    const auto accountsLink = Link(Link::Reconnect, QString());
+    const auto curUser = getApp()->accounts->twitch.getCurrent();
+    const auto bannedText = confirmedBan
+                                ? QString("You were banned from this channel!")
+                                : QString("You were forced off this channel.");
+
+    const auto reconnectPromptText =
+        confirmedBan
+            ? QString(
+                  "If you believe you have been unbanned, try reconnecting.")
+            : QString("Try reconnecting.");
+
+    auto builder = MessageBuilder();
```

Comment:
```suggestion
    MessageBuilder builder;
```

Reply:
Applied in 1d19dea4

Reply:
Maybe it's not a good idea to make code reviews at 5am without sleeping because you can't be sure if it's you seeing things or @mm2pl applying suggested change in wrong place (at line 703).

Reply:
Good catch @zneix, you are definitely correct here.

@Mm2PL, could you apply the suggested change in the correct place? (And maybe also revert the change from 1d19dea?)

Reply:
will fix when i can